src/screens/chats/container/chatThreadContainer.tsx

```
@@ -715,14 +716,41 @@ export const ChatThreadContainer: React.FC<ChatThreadContainerProps> = ({
+              }
+            }
+            // JSON.parse can yield non-objects (null, primitives) — guard against
+            // `jsonMeta?.description` blowing up downstream.
+            if (!jsonMeta || typeof jsonMeta !== 'object') {
+              jsonMeta = {};
+            }
+            const postForExtract = { ...post, json_metadata: jsonMeta };
+            // Platform.OS is wider than 'ios' | 'android' (includes web/windows/macos);
+            // narrow safely so postBodySummary always gets a value it accepts.
+            const summaryPlatform: 'ios' | 'android' =
+              Platform.OS === 'ios' ? 'ios' : 'android';
```

ESLint

[error] Delete `⏎·············`

(prettier/prettier)